src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -107,15 +119,16 @@ def __init__(self, mgr: "CephadmOrchestrator"):
 
     def make_daemon_spec(self, host: str,
                          daemon_id: str,
-                         netowrk: str,
+                         network: str,
                          spec: ServiceSpecs,
                          daemon_type: Optional[str] = None,) -> CephadmDaemonSpec:
         return CephadmDaemonSpec(
             host=host,
             daemon_id=daemon_id,
             spec=spec,
-            network=netowrk,
-            daemon_type=daemon_type
+            network=network,
+            daemon_type=daemon_type,
+            resources=spec.resources,
```

Comment:
I would like to keep the resources within the spec. :-1: for separating it, except we **really** want to deploy individual daemons of a services with specific memory limits.